packages/enabled-mk/0.1.0/README.md

```diff
@@ -0,0 +1,44 @@
+### `enabled-mk`, an espanso plugin
+
+If Espanso has become as ingrained in your every day life as it has for mine, then `enabled-mk` is for you.
+Simply type `:mk` or `;mk` to trigger the associated form menu.
+
+Under the hood, a Python one-liner takes Stdin arguments and uses them to interact with the espanso config files. Namely, it does this to deserialize, update, then re-serialize the data back into `espanso/match/base.yml`.
+
+Additionally, this plugin supports Vim users who accidentally encounter <kbd>:</kbd>-prefixed patterns and want a suitable
+alternative. To other authors -- please consider enabling <kbd>(;|:)</kbd> regex syntax instead of a static <kbd>:</kbd>
```

Comment:
Thank you for explaining why some people prefer `;` rather than `:` prefixes, apart from the reduction in the number of keypresses!
Federico [warned](https://espanso.org/docs/matches/regex-triggers/#when-not-to-use-regex-triggers) against over-use of regex triggers - I'm not sure how big a problem it would be though. At least converting "` :`" to "` ;`", or vice versa, is a trivial "search-and-replace" for those who prefer the latter.